### PR TITLE
Add a link in README to bender known variables.

### LINKED ISSUE #211: Documentation about known variables
Hello,

I was fiddling with `ab` to find how to create a playbook to build a container and failed to find the known variables bender use (I have a vague idea looking at the example in the `README` file though.

Is it possible to add them, or at least provide it here and I'll do a PR to add them to the `README` file.

Thanks

### PR DESCRIPTION
**Pull request by bmillemathias**
SSIA

fixes #211

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +274,8 @@ This is how the playbook looks:
 
 As you can see, the whole build processed is configured by the variable
 `ansible_bender`.
+The list of known variables by ansible-bender is detailled in the document
```
Comment (kmehant) [inline README.md]:
```suggestion
The list of known variables by ansible-bender is detailed in the document
```

Comment (bmillemathias) [reply README.md]: Thansk for spotting the type

**Comment by TomasTomecek** [comment]
@bmillemathias thank you for the contribution and @kmehant for the review

looks like CI is having hard times

**Pull request merged**